### PR TITLE
feat: multiple updates and enhancements

### LINKED ISSUE #3977: Spelling Mistake in proc_access_win_invoke_phantom.yml
Title has the word Ascess in place of Access.

### PR DESCRIPTION
**Pull request by nasbench**
This PR covers the following:

- Resolves #3977
- Deprecate fd6e2919-3936-40c9-99db-0aa922c356f7 in favor of a more generic rule that focuses on the encoded version of "Invoke-". This rule existed before and was also a high-level rule. It doesn't make sense to keep the specific one when they have the same severity and they're describing the "same" type of threat which is encoded "malicious" cmdlet.
- Deprecate 47688f1b-9f51-4656-b013-3cc49a166a36 in favor of a new generic rule that focuses on encoded WMI classes. 
- Deprecate 5b572dcf-254b-425c-a8c5-d9af6bea35a6 and merge it in bb780e0c-16cf-4383-8383-1e5471db6cf9
- Add other quality of life improvements to older rules including: 
  -  Updated title
  -  Add related fields
  -  Add `OriginalFileName` (where it make sense)
  -  General metadata updates
  -  FP tuning of some rules

### CODE REVIEW

File: rules/windows/process_creation/proc_creation_win_powershell_base64_frombase64string.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
-title: Encoded FromBase64String
+title: PowerShell Base64 Encoded FromBase64String Keyword
 id: fdb62a13-9a81-4e5c-a38f-ea93a16f6d7c
 status: test
-description: Detects a base64 encoded FromBase64String keyword in a process command line
+description: Detects usage of a base64 encoded "FromBase64String" cmdlet in a process command line
 author: Florian Roth
 date: 2019/08/24
 modified: 2022/03/07
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_powershell_base64_frombase64string.yml:7]:
```suggestion
modified: 2023/01/31
```

File: rules/windows/process_creation/proc_creation_win_powershell_base64_reflective_assembly_load.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ description: Detects base64 encoded .NET reflective loading of Assembly
 references:
     - https://github.com/Neo23x0/Raccine/blob/20a569fa21625086433dcce8bb2765d0ea08dcb6/yara/mal_revil.yar
     - https://thedfirreport.com/2022/05/09/seo-poisoning-a-gootloader-story/
-author: "Christian Burkard, pH-T"
+author: Christian Burkard, pH-T
 date: 2022/03/01
 modified: 2022/05/20
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_powershell_base64_reflective_assembly_load.yml:13]:
```suggestion
modified: 2023/01/30
```

File: rules/windows/process_creation/proc_creation_win_powershell_xor_commandline.yml [hunk 3]
```diff
@@ -23,25 +32,20 @@ detection:
...
+        CommandLine|contains: 'bxor'
+    selection_cli_other:
         CommandLine|contains:
-            - 'bxor'
+            - 'ForEach'
+            - 'for('
+            - 'for '
             - '-join '
-            - '-join'''
+            - "-join'"
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_powershell_xor_commandline.yml:43]: `'-join'''` is the yaml to escape `'` in a string.
https://yaml.org/spec/1.2.2/#57-escaped-characters
```
7.3.2. Single-Quoted Style

The single-quoted style is specified by surrounding “'” indicators. Therefore, within a single-quoted scalar, such characters need to be repeated. This is the only form of escaping performed in single-quoted scalars. In particular, the “\” and “"” characters may be freely used. This restricts single-quoted scalars to [printable](https://yaml.org/spec/1.2.2/#character-set) characters. In addition, it is only possible to break a long single-quoted line where a [space](https://yaml.org/spec/1.2.2/#white-space-characters) character is surrounded by non-[spaces](https://yaml.org/spec/1.2.2/#white-space-characters).
```

File: rules/windows/process_creation/proc_creation_win_reg_lsa_ppl_protection_disabled.yml [hunk 4]
```diff
@@ -1,21 +1,23 @@
-title: Registry Disabling LSASS PPL
+title: LSA PPL Protection Disabled Via Reg
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_reg_lsa_ppl_protection_disabled.yml:1]:
```suggestion
title: LSA PPL Protection Disabled Via Reg.EXE
```
You did it like that in another rule. I think in this case .EXE makes it more readable/understandable

File: rules/windows/process_creation/proc_creation_win_reg_lsa_ppl_protection_disabled.yml [hunk 5]
```diff
@@ -1,21 +1,23 @@
...
 id: 8c0eca51-0f88-4db2-9183-fdfb10c703f9
 status: experimental
-description: Detects reg command lines that disables PPL on the LSA process
+description: Detects usage of the "reg.exe" utility to disable PPL protection on the LSA process
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_reg_lsa_ppl_protection_disabled.yml:4]:
```suggestion
description: Detects the usage of the "reg.exe" utility to disable PPL protection on the LSA process
```

File: rules-deprecated/windows/proc_creation_win_powershell_base64_invoke_susp_cmdlets.yml [hunk 6]
```diff
@@ -1,14 +1,12 @@
 title: Malicious Base64 Encoded Powershell Invoke Cmdlets
 id: fd6e2919-3936-40c9-99db-0aa922c356f7
-related:
-    - id: 6385697e-9f1b-40bd-8817-f4a91f40508e
-      type: similar
-status: test
```
[the remark this answers is not in the document]
Comment (nasbench) [reply rules-deprecated/windows/proc_creation_win_powershell_base64_invoke_susp_cmdlets.yml]: I've reverted back the related fields.

As for the argument about deprecating the Cmdlet rule. It's just about the cost of maintenance as we've talked about internally :D

Note for future readers: If a lot of FPs are found with the new logic. We'll create a specific rules for highly mal/susp encoded cmdlets

File: rules/windows/process_creation/proc_creation_win_powershell_xor_commandline.yml (hunk 3 above)
Comment (frack113) [reply rules/windows/process_creation/proc_creation_win_powershell_xor_commandline.yml:43]: the easy way is not always the answer

**Review by frack113 — APPROVED** [review]

**Pull request merged**